### PR TITLE
a third banner ad is bourned

### PR DESCRIPTION
**Pull request by ChillyRelleno**
ayp_atg

### CODE REVIEW

**Comment by sshirokov** [comment]
Please add the banner image to the description.

File: lib/app.coffee [hunk 1]
```diff
@@ -38,8 +38,10 @@ app.engine 'handlebars', handlebars
       ads = [
         '<a href="http://amongthechosen.com/" title="Among the Chosen"><img alt="Ad for Among the Chosen" src="/static/images/ayp_atc.jpg"></a>',
         '<a href="http://19teeth.org/" title="19 Tooth Foundation"><img alt="Ad for 19 Tooth Foundation" src="/static/images/ayp_19t.jpg"></a>',
+		'<a href="https://www.youtube.com/channel/UCke_95iHaMHXxzD13TpCR4Q" title="Arctic Taco Gaming"><img alt="Ad for Arctic Taco Gaming" src="/static/images/ayp_atg.png"></a>',
       ]
-      ad = if Math.random() > 0.5 then ads[0] else ads[1]
+      #ad = if Math.random() > 0.5 then ads[0] else ads[1]
```
Comment (sshirokov) [inline lib/app.coffee:43]: If you're replacing the code, you don't need the comment, you just replace the code.

File: lib/app.coffee [hunk 2]
```diff
@@ -38,8 +38,10 @@ app.engine 'handlebars', handlebars
...
+	  ad = ads[Math.floor(math.random()*3)]
```
Comment (sshirokov) [inline lib/app.coffee:44]: Why did the indentation change?

Comment (sshirokov) [inline lib/app.coffee:44]: Now that there are not two, that constant `3` should instead be the actual length of `ads`

**Review by sshirokov — CHANGES_REQUESTED** [review]

File: lib/app.coffee (hunk 2 above)
Comment (sshirokov) [reply lib/app.coffee:44]: Also, I'm not gonna do the math, but I'm pretty sure `Math.floor()` isn't as good of a distribution as `Math.round`

File: lib/app.coffee [hunk 3]
```diff
@@ -38,8 +38,10 @@ app.engine 'handlebars', handlebars
       ads = [
         '<a href="http://amongthechosen.com/" title="Among the Chosen"><img alt="Ad for Among the Chosen" src="/static/images/ayp_atc.jpg"></a>',
         '<a href="http://19teeth.org/" title="19 Tooth Foundation"><img alt="Ad for 19 Tooth Foundation" src="/static/images/ayp_19t.jpg"></a>',
+		'<a href="https://www.youtube.com/channel/UCke_95iHaMHXxzD13TpCR4Q" title="Arctic Taco Gaming"><img alt="Ad for Arctic Taco Gaming" src="/static/images/ayp_atg.png"></a>',
```
Comment (sshirokov) [inline lib/app.coffee:41]: Indentation wtf here as well.

**Review by sshirokov — CHANGES_REQUESTED** [review]

File: lib/app.coffee (hunk 2 above)
Comment (ChillyRelleno) [reply lib/app.coffee:44]: Tabs vs spaces.

File: lib/app.coffee (hunk 3 above)
Comment (ChillyRelleno) [reply lib/app.coffee:41]: why is it like this

File: lib/app.coffee [hunk 4]
```diff
@@ -38,8 +38,9 @@ app.engine 'handlebars', handlebars
       ads = [
         '<a href="http://amongthechosen.com/" title="Among the Chosen"><img alt="Ad for Among the Chosen" src="/static/images/ayp_atc.jpg"></a>',
         '<a href="http://19teeth.org/" title="19 Tooth Foundation"><img alt="Ad for 19 Tooth Foundation" src="/static/images/ayp_19t.jpg"></a>',
+        '<a href="https://www.youtube.com/channel/UCke_95iHaMHXxzD13TpCR4Q" title="Arctic Taco Gaming"><img alt="Ad for Arctic Taco Gaming" src="/static/images/ayp_atg.png"></a>',
       ]
-      ad = if Math.random() > 0.5 then ads[0] else ads[1]
+      ad = ads[Math.floor(math.random()*ads.length)]
```
Comment (sshirokov) [inline lib/app.coffee:43]:
```suggestion
      ad = ads[Math.floor(Math.random()*ads.length)]
```

`Math` is capitalized

Comment (sshirokov) [reply lib/app.coffee:43]: And I'm guessing that you don't care if the distribution is even? I'm also not sure nor do I know if it matters.

**Review by sshirokov — CHANGES_REQUESTED** [review]

**Review by ChillyRelleno — COMMENTED** [review]
looks like decent c to me!

**Review by sshirokov — APPROVED** [review]

**Pull request merged**